src/model/BookingData.java

```diff
@@ -17,8 +17,14 @@ public BookingData(LocalDateTime entryDate, LocalDateTime departureDate, Payment
 		this.paymentMethod=paymentMethod;
 	}
 	
-	public BookingData(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethod paymentMethod,
-			BigDecimal bookingPrice) {
+	public BookingData(LocalDateTime entryDate, LocalDateTime departureDate, BigDecimal bookingPrice,
```

Comment:
This constructor is repeated, please remove the old one, or use the new one:

The old one
```java
BookingData(LocalDateTime entryDate, LocalDateTime departureDate, PaymentMethod paymentMethod,BigDecimal bookingPrice)
```

The new one:
```java
BookingData(LocalDateTime entryDate, LocalDateTime departureDate, BigDecimal bookingPrice,
			PaymentMethod paymentMethod)
```